<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeneralSettingsPanel</class>
 <widget class="QWidget" name="GeneralSettingsPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>359</width>
    <height>263</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Auto-Save</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="toolTip">
         <string>Path where temporal auto-save files are stored</string>
        </property>
        <property name="text">
         <string>Auto-save Path:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="label_2">
        <property name="toolTip">
         <string>Time interval at which auto-saves are performed</string>
        </property>
        <property name="text">
         <string>Auto-save Interval:  </string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="autosavePath">
        <property name="toolTip">
         <string>Path where temporal auto-save files are stored</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QSpinBox" name="autosaveInterval">
        <property name="toolTip">
         <string>Time interval at which auto-saves are performed</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="suffix">
         <string> min</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>60</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Segmentation</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="traceabilty">
        <property name="toolTip">
         <string>Store all information needed to repeat the analysis inside the .seg files</string>
        </property>
        <property name="text">
         <string>Save traceability information</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label">
          <property name="toolTip">
           <string/>
          </property>
          <property name="text">
           <string>User Name:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="userName">
          <property name="toolTip">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>245</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
